microservice-app/src/main/java/com/sdevelopment/skt/microservice/dao/impl/ProductRepositoryImpl.java

```diff
@@ -14,18 +16,30 @@
     @PersistenceContext
     private EntityManager em;
 
-    public void saveProduct(Product product) {
-        StoredProcedureQuery saveProductProcedure = em.createNamedStoredProcedureQuery("saveProduct");
-        saveProductProcedure.setParameter("_name", product.getName());
+    //@Transactional
```

Comment:
Avoid commented code. Also, why is (or is not) required to be @Transactional? What is the purpose of this annotation?

Reply:
I included it because I was having some issues with the entity manager. I have to check the theory but I think is not mandatory in this case because JPA and Hibernate handle the transactions.

Reply:
Ok, I suggest that if it's not mandatory, you may remove it